src/main/java/com/denizenscript/depenizen/bukkit/commands/worldedit/WorldEditCommand.java

```diff
@@ -0,0 +1,363 @@
+    //
+    // @Tags
+    // <PlayerTag.we_selection>
+    //
+    // @Usage
+    // Use to save a cuboid to a schematic.
+    // - worldedit cuboidschematic file:<[filepath]> cuboid:<player.we_selection> position:<player.location>
+    //
+    // @Usage
+    // Use to copy a cuboid to a player's clipboard.
+    // - worldedit cuboidclipboard cuboid:<player.we_selection> position:<player.location>
+    //
+    // @Usage
+    // Use to load a schematic into a player's clipboard.
+    // - worldedit schematicclipboard file:<[filepath]>
+    //
+    // @Usage
+    // Use to paste a schematic at a location
+    // - worldedit schematicpaste file:<[filepath]> position:<player.location>
+    //
+    // @Usage
+    // Use to paste a schematic at a location with a player attached to the edit history.
+    // - worldedit schematicpaste file:<[filepath]> position:<player.location> undoable noair rotate:90
+    //
+    // -->
+
+    private enum Action {CUBOIDSCHEMATIC, CUBOIDCLIPBOARD, SCHEMATICPASTE, SCHEMATICCLIPBOARD}
+
+    @Override
+    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
+
+        for (Argument arg : scriptEntry.getProcessedArgs()) {
+
+            if (!scriptEntry.hasObject("position")
+                    && arg.matchesPrefix("position")) {
+                scriptEntry.addObject("position", arg.asType(LocationTag.class));
+            }
+
+            else if (!scriptEntry.hasObject("file")
+                    && arg.matchesPrefix("file")) {
+                scriptEntry.addObject("file", arg.asElement());
+            }
+
+            else if (!scriptEntry.hasObject("cuboid")
+                    && arg.matchesPrefix("cuboid")) {
+                scriptEntry.addObject("cuboid", arg.asType(CuboidTag.class));
+            }
+
+            else if (!scriptEntry.hasObject("noair")
+                    && arg.matches("noair")) {
+                scriptEntry.addObject("noair", new ElementTag(true));
+            }
+
+            else if (!scriptEntry.hasObject("undoable")
+                    && arg.matches("undoable")) {
+                scriptEntry.addObject("undoable", new ElementTag(true));
+            }
+
+            else if (!scriptEntry.hasObject("rotate")
+                    && arg.matchesPrefix("rotate")) {
+                scriptEntry.addObject("rotate", arg.asElement());
+            }
+
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesEnum(Action.values())) {
+                scriptEntry.addObject("action", arg.asElement());
+            }
+
+            else {
+                arg.reportUnhandled();
+            }
+
+        }
+        if (!scriptEntry.hasObject("action")) {
+            throw new InvalidArgumentsException("Action not specified!");
+        }
+        if (Utilities.entryHasPlayer(scriptEntry)) {
+            scriptEntry.addObject("target", Utilities.getEntryPlayer(scriptEntry));
```

Comment:
What is this? If it's not user-specifiable, it should not be a script entry object...

Reply:
Please do not use Depenizen code as a reference. Use Denizen as a reference.